### PR TITLE
Add bslmeval scripts

### PR DESCRIPTION
**Pull request by Muennighoff**

### CODE REVIEW

File: evaluation/results/tr13/merge_all_json.py [hunk 1]
```diff
@@ -50,14 +50,30 @@ def main():
         with open(json_file, "r") as fi:
             data = json.load(fi)
 
-        key = f"{data['dataset_name']}_{data['dataset_config_name']}"
-        if key in results:
-            assert data["template_name"] not in results
-            results[key][data["template_name"]] = data
+        if str(json_file.name).startswith("slim"):
```
Comment (thomasw21) [inline evaluation/results/tr13/merge_all_json.py:53]: Can we just test is "results" is a key or not?

File: evaluation/results/tr13/merge_all_json.py [hunk 2]
```diff
@@ -50,14 +50,30 @@ def main():
...
+            print(f"Parsing {json_file} as bigscience/lm-eval-harness file.")
+            for dic in data["results"]:
+                key = f'{dic["task_name"]}+{dic["prompt_name"]}'
+                results.setdefault(key, {})
+                results[key] = {
+                    **results[key], 
+                    **{subk: subv for subk, subv in dic.items() if type(subv) in [int, float]}
```
Comment (thomasw21) [inline evaluation/results/tr13/merge_all_json.py:60]: Are there actual results that are neither `int` or `float`?

**Review by thomasw21 — CHANGES_REQUESTED** [review]

File: evaluation/results/tr13/download_bslmeval.slurm [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+python3 main.py --model hf-causal \
```
[the remark this answers is not in the document]
Comment (Muennighoff) [reply evaluation/results/tr13/download_bslmeval.slurm:31]: True, but I wanted to allow easy copy-pasting of the tasks. 
If I use your solution I need to rewrite `wmt14_fr_en` as `("wmt14", "fr_en")`. 
This script guarantees that rerunning with the exact same task names will work 😁

File: evaluation/results/tr13/download_bslmeval.slurm [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    --model_args pretrained=hf-internal-testing/tiny-random-gpt2,use_accelerate=True,tokenizer=hf-internal-testing/tiny-random-gpt2,dtype=float16 \
+    --tasks wmt14_fr_en,wmt19_ru_en,wmt19_zh_en \
+    --device cuda \
```
[the remark this answers is not in the document]
Comment (Muennighoff) [reply evaluation/results/tr13/download_bslmeval.slurm:34]: Doesn't [prepost](http://www.idris.fr/eng/jean-zay/cpu/jean-zay-cpu-hw-eng.html#prepost) always have a GPU?

**Pull request merged**